docs/book.toml

```diff
@@ -0,0 +1,15 @@
+[book]
+authors = ["Concord Authors"]
+language = "en"
+src = "src"
+title = "Concord"
+
+[preprocessor.variables.variables.site]
+concord_core_version = "2.32.0"
```

Comment:
Can be replaced with `${project.version}` value, if we wire up the mdbook build through Maven.